docs/getting-help/getting-help.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Getting Help with Prophecy
+sidebar_position: 3
+id: getting-help-with-prophecy
+description: Getting Help with Prophecy
+tags: []
+---
+
+At Prophecy, we're dedicated to ensuring you have the support you need, whether you're exploring our platform during a trial or you're one of our valued customers.
+
+## For Trial Users
+
+- **Get Trial Support in Slack:** Have questions during your trial? Join the conversation in the #support channel on our Slack community. Our dynamic community is here to assist you. [Join #support on Slack](https://prophecy-io-support.slack.com/archives/C01P1PD7JJY).
+- _If you're a Trial customer and unable to signup for our Slack Community, please reach out to Sales@Prophecy.io for quick assistance._
```

Comment:
should we also say that they can reach out to sales@prophecy if they require product demo from sales team to get a clear picture.